plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotSessionProperties.java

```diff
@@ -84,9 +86,19 @@ public PinotSessionProperties(PinotConfig pinotConfig)
                         COUNT_DISTINCT_PUSHDOWN_ENABLED,
                         "Enable count distinct pushdown",
                         pinotConfig.isCountDistinctPushdownEnabled(),
+                        false),
+                stringProperty(
+                        QUERY_OPTIONS,
+                        "Pinot query option in the format of key:value,key:value",
+                        pinotConfig.getQueryOptions(),
                         false));
     }
 
+    public static String getQueryOptions(ConnectorSession session)
```

Comment:
Move to the bottom for the same order as fields. Also, please return `Optinal<String>`. We should avoid returning null as much as possible.